neptune/new/attribute_container.py

```diff
@@ -0,0 +1,461 @@
+#
```

Comment:
Can it be internal?

Reply:
This was my initial thought, but it implements a lot of public methods (assign, fetch etc.) -- and I don't really want it hidden in internal class.